src/app/HeaderContext.jsx

```diff
@@ -0,0 +1,28 @@
+'use client'
+
+import { createContext, useContext, useState } from 'react';
+
+const MobileMenuContext = createContext();
+
+export const useMobileMenuContext = () => {
+  return useContext(MobileMenuContext);
+};
+
+export const MobileMenuProvider = ({ children }) => {
```

Comment:
File called HeaderContext, should be MobileMenuProvider

Also should be moved to components folder

Reply:
fixed